<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Файловый менеджер</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Intro;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>281</height>
       </rect>
      </property>
      <widget class="QPushButton" name="createrFileButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Создать документ</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboTasksBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboFormatBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
